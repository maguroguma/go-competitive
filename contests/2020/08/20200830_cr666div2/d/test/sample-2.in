6
3
3 2 2
3
2 2 2
2
2 2
2
2 1
4
2 2 2 2
4
3 2 2 2
