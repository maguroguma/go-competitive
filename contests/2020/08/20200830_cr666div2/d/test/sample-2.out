T
HL
HL
T
HL
T
